docs/griptape-framework/structures/tasks.md

```diff
@@ -63,7 +63,7 @@ agent.run("How do I bake a cake?")
 
 ## Prompt Task
 
-For general purpose prompting, you can use the [PromptTask](../../reference/griptape/tasks/prompt_task.md):
+For general purpose prompting, you can use the [PromptTask](../../griptape-framework/structures/tasks.md):
```

Comment:
This should be reverted. Make sure you follow the instructions outlined [here](https://docs.griptape.ai/contributing/#pull-requests)

Reply:
ok!